cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -157,7 +161,7 @@ def test_reports_create_schedule_send_report(smtp_test, schedule):
             1. Queueing the schedule must send the report via email to all the users.
     """
     schedule.queue()
-    emails_sent = ",".join(schedule.emails)
+    emails_sent = ",".join(schedule.email.get("to_emails"))
```

Comment:
This will cause a `TypeError`, the default for the `get` call has to be an iterable.

Reply:
I am not sure I understand what you're saying. Are you saying  I should assign a default value to the `get` call?
`schedule.email` is an iterable and so is `to_emails` and both are non-empty iterables, so at least for this case, I don't think we would be getting any TypeError.

Reply:
correct, the `get()` call needs an iterable as its default, in case 'to_emails' isn't set.